lib/cli/interactive-setup/aws-credentials.js

```diff
@@ -3,15 +3,99 @@
 const chalk = require('chalk');
 const _ = require('lodash');
 const inquirer = require('@serverless/utils/inquirer');
+const memoizee = require('memoizee');
```

Comment:
Does it cover changes from other PR? If so it might be good to temporarily base it against other PR branch, otherwise it's hard to review

Reply:
Correct, it's due to the fact that it depends on changes included in https://github.com/serverless/serverless/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc